# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cinelerra/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- cinelerra-2.0/guicast/bclistbox.h_orig	2006-03-04 17:26:23.036254456 +0100
+++ cinelerra-2.0/guicast/bclistbox.h	2006-03-04 17:28:14.638288368 +0100
@@ -351,7 +351,7 @@
 
 	int get_title_h();
 	int calculate_item_coords();
-	void BC_ListBox::calculate_last_coords_recursive(
+	void calculate_last_coords_recursive(
 		ArrayList<BC_ListBoxItem*> *data,
 		int *icon_x,
 		int *next_icon_x,
--- cinelerra-2.0/cinelerra/file.h_orig	2006-03-04 17:30:09.579814600 +0100
+++ cinelerra-2.0/cinelerra/file.h	2006-03-04 17:30:49.394761808 +0100
@@ -140,7 +140,7 @@
 
 
 // Read frame of video into the argument
-	int File::read_frame(VFrame *frame);
+	int read_frame(VFrame *frame);
 
 
 // The following involve no extra copies.
--- cinelerra-2.0/cinelerra/virtualconsole.h_orig	2006-03-04 17:33:35.245548656 +0100
+++ cinelerra-2.0/cinelerra/virtualconsole.h	2006-03-04 17:34:37.513082552 +0100
@@ -28,7 +28,7 @@
 
 // Called during each process buffer operation to reset the status
 // of the attachments to unprocessed.
-	void VirtualConsole::reset_attachments();
+	void reset_attachments();
 	void dump();
 
 
--- cinelerra-2.0/cinelerra/zoombar.h_orig	2006-03-04 17:35:21.209439696 +0100
+++ cinelerra-2.0/cinelerra/zoombar.h	2006-03-04 17:35:39.708627392 +0100
@@ -24,7 +24,7 @@
 	~ZoomBar();
 
 	int create_objects();
-	void ZoomBar::resize_event();
+	void resize_event();
 	int draw();
 	int resize_event(int w, int h);
 	void redraw_time_dependancies();
--- cinelerra-2.0/cinelerra/framecache.h_orig	2006-03-04 17:36:51.327739632 +0100
+++ cinelerra-2.0/cinelerra/framecache.h	2006-03-04 17:37:30.361805552 +0100
@@ -73,7 +73,7 @@
 		int64_t position,
 		double frame_rate,
 		int *item_return);
-	int FrameCache::frame_exists(int64_t position, 
+	int frame_exists(int64_t position, 
 		double frame_rate,
 		int color_model,
 		int w,
--- cinelerra-2.0/cinelerra/renderfarmclient.h_orig	2006-03-04 17:39:37.213521160 +0100
+++ cinelerra-2.0/cinelerra/renderfarmclient.h	2006-03-04 17:40:15.076765072 +0100
@@ -70,7 +70,7 @@
 
 
 
-	void RenderFarmClientThread::read_preferences(int socket_fd, 
+	void read_preferences(int socket_fd, 
 		Preferences *preferences);
 	void read_asset(int socket_fd, Asset *asset);
 	void read_edl(int socket_fd, 
--- cinelerra-2.0/cinelerra/vrender.h_orig	2006-03-04 17:41:52.474958296 +0100
+++ cinelerra-2.0/cinelerra/vrender.h	2006-03-04 17:42:15.841406056 +0100
@@ -48,7 +48,7 @@
 // Flash the output on the display
 	int flash_output();
 // Determine if data can be copied directly from the file to the output device.
-	void VRender::get_render_strategy(Edit* &playable_edit, 
+	void get_render_strategy(Edit* &playable_edit, 
 		int &colormodel, 
 		int &use_vconsole,
 		int64_t position);
